application/common/models/Mfa.php

```diff
@@ -300,7 +369,7 @@ public static function create(int $userId, string $type, string $label = null, s
         $existing = self::findOne(['user_id' => $userId, 'type' => $type, 'verified' => 1]);
```

Comment:
Now that the mfa_webauthn table has been split out, will `$existing` ever be an Mfa of type `TYPE_WEBAUTHN`, or will `verified` always be `null` for those records in the `mfa` table?